integration/aggregate_stats_compat_test.go

```diff
@@ -25,8 +25,8 @@ import (
 	"github.com/FerretDB/FerretDB/integration/shareddata"
 )
 
-func TestAggregateCompatCollStats(t *testing.T) {
-	t.Parallel()
+func TestAggregateCompatCollStats(tt *testing.T) {
```

Comment:
This test fails on CI. Actually, does this test work on your local environment @noisersup :thinking: ?

```
2023-08-02T15:42:00.5828659Z --- FAIL: TestAggregateCompatCollStats (0.00s)
2023-08-02T15:42:00.5828912Z     --- FAIL: TestAggregateCompatCollStats/CountAndStorageStats (0.43s)
2023-08-02T15:42:00.5829093Z         aggregate_stats_compat_test.go:68: 
2023-08-02T15:42:00.5829508Z             	Error Trace:	/home/runner/work/FerretDB/FerretDB/integration/setup/listener.go:152
2023-08-02T15:42:00.5830152Z             	            				/home/runner/work/FerretDB/FerretDB/integration/setup/setup_compat.go:95
2023-08-02T15:42:00.5830845Z             	            				/home/runner/work/FerretDB/FerretDB/integration/aggregate_stats_compat_test.go:77
2023-08-02T15:42:00.5831057Z             	Error:      	Received unexpected error:
2023-08-02T15:42:00.5832122Z             	            	[registry.go:76 metadata.NewRegistry] [registry.go:92 metadata.(*Registry).loadCollections] SQL logic error: no such table: _ferretdb_collections (1)
2023-08-02T15:42:00.5832424Z             	Test:       	TestAggregateCompatCollStats/CountAndStorageStats
```